tests/integration/tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+"""
+Integration tests setup configurations and fixtures
+"""
+import time
+from pathlib import Path
+import pytest
+from commonlib.io_utils import get_k8s_yaml_objects
+
+
+DEPLOY_YML = "../../deploy/cloudbeat-pytest.yml"
+
+
+@pytest.fixture(scope='module', name='start_stop')
+def fixture_start_stop(k8s, api_client, cloudbeat_agent):
+    """
+    This fixture starts cloudbeat/agent on test module setup and
+    stops on teardown of the test module
+    @param k8s: Kubernetes client object
+    @param api_client: Docker api / FileSystem api
+    @param cloudbeat_agent: Cloudbeat configuration
+    @return: Kubernetes object, Api client, Cloudbeat config
+    """
+    file_path = Path(__file__).parent / DEPLOY_YML
+    if k8s.get_agent_pod_instances(agent_name=cloudbeat_agent.name,
+                                   namespace=cloudbeat_agent.namespace):
+        k8s.delete_from_yaml(get_k8s_yaml_objects(file_path=file_path))
+    k8s.start_agent(yaml_file=file_path, namespace=cloudbeat_agent.namespace)
+    time.sleep(5)
```

Comment:
we should avoid sleep in general, can you find an alternative?

Reply:
sleep replaced by waiting for resource function

Reply:
@gurevichdmitry there's still sleep here?

Reply:
no, may be you check outdated file